exercises/practice/two-bucket/.meta/gen.go

```diff
@@ -50,13 +50,20 @@ type bucketTestCase struct {
 	goalBucket	string
 	moves		int
 	otherBucket	int
-	errorExpected   bool // always false for generated test cases.
+	expectedError   string
 }
 
 var testCases = []bucketTestCase {
-{{range .J.Cases}}{
-	"{{.Description}}",
-	{{.Input.BucketOne}}, {{.Input.BucketTwo}}, {{.Input.Goal}}, "{{.Input.StartBucket}}", "{{.Expected.GoalBucket}}", {{.Expected.Moves}}, {{.Expected.OtherBucket}}, false,
+{{range .J.measure}}{
+		description:   	{{ printf "%q" .Description}},
+		bucketOne:     	{{ printf "%d" .Input.BucketOne}},
+		bucketTwo:      {{ printf "%d" .Input.BucketTwo}},
+		goal:           {{ printf "%d" .Input.Goal}},
+		startBucket:   	{{ printf "%q" .Input.StartBucket}},
+		goalBucket:    	{{ printf "%q" .Expected.GoalBucket}},
+		moves:        	{{ printf "%q" .Expected.Moves}},
+		otherBucket:   	{{ printf "%q" .Expected.OtherBucket}},
```

Comment:
```suggestion
		moves:        	{{ printf "%d" .Expected.Moves}},
		otherBucket:   	{{ printf "%d" .Expected.OtherBucket}},
```

This is making the test case literals appear as hexadecimal.